src/components/sections/ProductGallery/ProductGalleryPage.tsx

```diff
@@ -37,6 +46,33 @@ function GalleryPage({ page, display, title, fallbackData }: Props) {
     [productList]
   )
 
+  /* 
+  PS.: Just for  the sake of demonstrantion, productSponsoredList was created by copying the data from
+  productList, and an arbitrary page (20) was choosen to get different products.
+  TODO: Change the productSponsoredList to get the products from a specific hook that return Sponsored Products,
+  or receive these products as props.
+  */
```

Comment:
Good idea, I think we should think about how to handle it to avoid getting lots of unused data and decrease our performance on Lighthouse. I'll leave a comment on the Jira task.